### DISCUSSION TITLE
Some icons not shown as correct icon

### DISCUSSION THREAD

**Discussion by cconvey** [lsd-rs/lsd#897] [Q&A]
What's the best forum for an lsd newbie seeking help?

(I'm setting it up for the first time on Gnome 3.44.0.  `echo $'\uf115'` shows me the folder icon as expected, but for a few file types I'm seeing what I assume are Chinese characters.)

**Comment by zwpaper** [comment]
Hi @cconvey, this is mainly the font problems, you could try another nerd font or specify the icons in icon theme.
